cmd/project/project_dump.go

```diff
@@ -159,17 +159,21 @@ var projectDatabaseDumpCmd = &cobra.Command{
 			return dErr
 		}
 
-		if gzipEnabled {
-			output += ".gz"
-		}
+		var w io.Writer
+		if output == "-" {
```

Comment:
Do you want maybe gzipped piped 🤔 . does that make sense 🤔

Reply:
What exactly do you mean? I doubt many people will use `--output -` with `--gzip` or `--zstd` but it should still work and it does with the current code.

Reply:
nvm, looked wrong